python/ray/rllib/models/fcnet.py

```diff
@@ -8,31 +8,57 @@
 from ray.rllib.models.model import Model
 from ray.rllib.models.misc import normc_initializer
 
+USER_DATA_CONFIGS = [
+    "fcnet_tag",  # Optional tag for fcnets to allow for more than one
+    "shared_model", # Optional tag testing if a multiagent shared model is being created
+]
+
+
 
 class FullyConnectedNetwork(Model):
     """Generic fully connected network."""
 
     def _init(self, inputs, num_outputs, options):
         hiddens = options.get("fcnet_hiddens", [256, 256])
+        # For value functions for a multiagent environment this will be passed a list
+        if isinstance(hiddens[0], list):
```

Comment:
So the issue here is that fcnet_hiddens is used both to create the value function and the policies. Does it make sense to pass both an fcnet_hiddens and a multiagent_fcnet_hiddens to avoid this issue?

Reply:
Yeah, i'd go with the more explicit option of having `multiagent_fcnet_hiddens`. It could be confusing for the user to have fcnet_hiddens sometimes take a list.